ligand/templates/bias_browser.html

```diff
@@ -3,1331 +3,1698 @@
-        table.dataTable.compact thead td {
-            padding: 5px 10px 5px 0px;
-            width: 0px;
-        }
-
-        table.dataTable.compact thead tr.over_header th {
-            border-bottom: 1px solid #ccc;
-        }
-
-        table.dataTable.compact thead th.leftborder {
-            border-left: 1px solid;
-        }
-
-        table.dataTable.compact thead th.rightborder {
-            border-right: 1px solid;
-        }
-
-        .yadcf-filter-range {
-            max-width: 40px !important;
-            font-family: sans-serif;
-            font-size: 80%;
-            font-weight: bold;
-            min-width: 40px;
-        }
-
-        .clicked_button {
-            background-color: rgb(215, 215, 215);
-        }
-
-        #overlay {
-            top: 0px;
-            position: absolute;
-            background: #f8f8f8;
-            /*border: 1px solid #333;*/
-            -webkit-box-shadow: 5px 0 2px -2px #888;
-            box-shadow: 5px 0 2px -2px #888;
-        }
-
-        .container {
-            width: 90%;
-        }
-
-        .wrapper {
-            overflow-x: scroll;
-            overflow-y: hidden;
-        }
-
-        #overlay tbody tr {
-            background-color: #f8f8f8;
-        }
-
-        #count_row th {
-            font-size: 11px;
-            text-align: center;
-        }
-
-        .switch-field label:hover {
-            cursor: pointer;
-            /* width: 100%; */
-        }
-
-        .switch-field input:checked+label {
-            background-color: #337ab7;
-            /* #a5dc86; */
-            box-shadow: none;
-        }
-
-        .border-left {
-            padding-left: 3px;
-        }
-
-        .switch-field label:first-of-type {
-            border-radius: 4px 0 0 4px;
-
-        }
-
-        .switch-field label:last-of-type {
-            border-radius: 0 4px 4px 0;
-        }
-
-        /* This is just for CodePen. */
-
-        .name {
-            width: 100px;
-            text-overflow: ellipsis;
-            cursor: pointer;
-            word-break: break-all;
-            overflow: hidden;
-            white-space: nowrap;
-        }
-
-        .name:hover {
-            overflow: visible;
-            white-space: normal;
-
-        }
-
-        .toolbar_data_css {
-            width: 250px;
-            margin-right: auto;
-            margin-left: 0px;
-            /* border: 1px solid; */
-            font-family: "Lucida Grande", Tahoma, Verdana, sans-serif;
-            font-weight: normal;
-            font-size: 120%;
-            line-height: 1.625;
-        }
-
-        .toolbar_data_css.td {
-            height: 15px;
-            max-width: 150px;
-            overflow: hidden;
-            white-space: nowrap;
-            text-overflow: ellipsis;
-        }
-    </style>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/dt/jszip-2.5.0/dt-1.10.16/b-1.5.1/b-html5-1.5.1/datatables.min.css" />
```

Comment:
Have to be careful with this. Can probably affect much of the datatables and yadcf functionalities you're using.
Probably might be better to go back to what we have in static both for css and js.

Reply:
I used that import from cdn to import additional buttons for datatables as we do not have them in static